dist.ini

```diff
@@ -18,42 +18,7 @@ version_regexp  = ^(1.\d+)$   ; this is the default
 
 [OurPkgVersion]
 
-[Prereqs]
-Data::Buffer   = 0.04
-Compress::Zlib = 0
-Crypt::DSA     = 1.17
-File::HomeDir  = 0
-LWP::UserAgent = 0
-MIME::Base64   = 0
-Math::BigInt   = 0
-URI::Escape    = 0
-
-; RT#123452 Require at least one Random Source
-Bytes::Random::Secure  = 0
-
-; see https://github.com/btrott/Crypt-OpenPGP/pull/10
-Alt::Crypt::RSA::BigInt = 0
-
-; PGP2 compat.
-Crypt::IDEA = 0
-Digest::MD5 = 0
-
-; PGP5 compat.
-Crypt::DES_EDE3 = 0
-Digest::SHA     = 0
-
-; GnuPG compat.
-Crypt::CAST5_PP  = 0
-Crypt::RIPEMD160 = 0.05
-Crypt::Rijndael  = 0
-
-; Other ciphers.
-Crypt::Blowfish = 0
-Crypt::Twofish  = 0
-
-[Prereqs/TestRequires]
-Test::More      = 0
-Test::Exception = 0
+[Prereqs::FromCPANfile]
```

Comment:
See above comment, I normally get the cpanfile automatically generated by Dist::Zilla